### PR TITLE
[SDAN-397] - Adding filter parameters into url

### PR DESCRIPTION
**Pull request by akintolga**
- With this change selected filters and navigation is reflected in URL
- On reload of the page all those values are applied 
- Now when sharing a topic the url will have all the parameters of the topic

### CODE REVIEW

File: newsroom/topics/views.py [hunk 1]
```diff
@@ -54,6 +54,26 @@ def is_user_topic(topic_id, user_id):
     return False
 
 
+def get_topic_url(topic):
```
Comment (petrjasek) [inline newsroom/topics/views.py]: wondering if we should rather use `topic=<topic-id>` and handle it on server, if topics are editable the url would have to change, but if it was shared already it won't work

Comment (akintolga) [reply newsroom/topics/views.py]: We could use id too. Afaik user cannot edit a topic (except name) but they can delete and recreate of course. Then shared email won't be valid. I was inclined to use URL as one feedback from AAP few weeks back was to use URL for the dropdown filters in agenda (which I did local store).

Comment (petrjasek) [reply newsroom/topics/views.py]: I see, if the query is not editable it's ok

File: assets/wire/actions.js [hunk 2]
```diff
@@ -212,6 +213,9 @@ export function fetchItems() {
                 const state = getState();
                 updateRouteParams({
                     q: state.query,
+                    filter: get(state, 'search.activeFilter'),
```
Comment (mdhaman) [inline assets/wire/actions.js]: Different browser have different limits on saving data to history. `updateRouteParams` fails on firefox, ie, etc after hitting the limit. 
You would see data changing as dispatch for updating the store is before `updateRouteParams` but the URL stops changing.

Comment (akintolga) [reply assets/wire/actions.js]: Hi @mdhaman I think the problem is the sixe of the state being too big for firefox to handle so as you say it was hitting the limit. I made the change now to trim the `items` and `itemsById` which were the biggest chunk in state. Now we fetch once the hidtory is popped back.

**Pull request merged**